locales/en-US/app.ftl

```diff
@@ -96,6 +127,15 @@ Home--additional-content-content = You can <strong>drag and drop</strong> a prof
 Home--compare-recordings-info = You can also compare recordings. <a>Open the comparing interface.</a>
 Home--recent-uploaded-recordings-title = Recent uploaded recordings
 
+## IdleSearchField
+
+IdleSearchField--search-input =
+    .placeholder = Enter filter terms
+
+## JsTracerSettings
```

Comment:
What do you think of adding some more comments for this component to help the localizers, because it's not visible by default.